marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/entitySearch/getMinAndMaxPropertyValues.api

```diff
@@ -0,0 +1,21 @@
+{
+  "functionName": "getNumericFacetsRange",
+  "params": [
+    {
+      "name": "entityIRI",
```

Comment:
As long as this function name needs to be changed, let's change this name too - we know this will now be "entityTypeId".